### PR TITLE
1355 - BDD - Aquifer Add/Edit

### PR DESCRIPTION
**Pull request by NickPhura**
Code Changes/Updates
-------------------------------------------------------
Added aquifer add/edit tests.
- Updated aquifer IDs to be more consistent, to make use in testing easier.
- Updated unit tests accordingly.

Misc minor updates.
- Updated some test assertions to allow for better logging if they fail (same thing being asserted though).
- Updated some test comments

Trello
-----------------------------------------------------
https://trello.com/c/s1GPfdNY/1355-epicaquiferdata-functional-tests-for-aquifer-data-add-edit-2

Version
------------------------------------------------------
Rebased on latest version of 'release/1.46.2' as of Dec 10, 2018.

### CODE REVIEW

**Comment by NickPhura** [comment]
[EPIC#AquiferData | Functional Tests for Aquifer Data Add Edit |2| !!!](https://trello.com/c/s1GPfdNY/1355-epicaquiferdata-functional-tests-for-aquifer-data-add-edit-2)

File: functional-tests/src/test/groovy/pages/app/RegistryProfilePage.groovy [hunk 1]
```diff
@@ -18,28 +18,27 @@ package pages
...
 
   /**
    * Constructor.
    *
-   * Uses the well oeprator's first and last name to at check the page, as the title is based on the operator's name.
+   * Uses the well operator's first and last name to-at check the page, as the title is based on the operator's name.
    *
-   * @param String the well operator surname (required)
-   * @param String the well operator firstname (required)
+   * @param String the well operator surName (required)
```
Comment (Sybrand) [inline functional-tests/src/test/groovy/pages/app/RegistryProfilePage.groovy]: typo

File: app/frontend/test/unit/specs/aquifers/components/__snapshots__/View.spec.js.snap [hunk 2]
```diff
@@ -700,9 +700,9 @@ exports[`View Component View mode matches the snapshot 1`] = `
       <div
         class="col"
       >
-        <h5>
+        <h4>
```
Comment (Sybrand) [inline app/frontend/test/unit/specs/aquifers/components/__snapshots__/View.spec.js.snap]: Why h5 -> h4? Is it a visual thing? Accessibility? Or actually relating to the tests?

Comment (NickPhura) [reply app/frontend/test/unit/specs/aquifers/components/__snapshots__/View.spec.js.snap]: 2 reasons:
1.  I needed some way to target the page header, as part of the test (standard practice thus far to ensure you are on the correct page).
2.  All of the other pages use an 'h4' for the headers.  The aquifer pages were the only ones that deviated, and as a result, had multiple h5 tags, which made the (1) difficult.

File: functional-tests/src/test/groovy/pages/app/RegistryProfilePage.groovy (hunk 1 above)
Comment (NickPhura) [reply functional-tests/src/test/groovy/pages/app/RegistryProfilePage.groovy]: Fixed, once I commit here.

**Comment by NickPhura** [comment]
@Sybrand 
Just to clarify, is the build broken due to some change in this PR, or just due to other pipeline issues, etc.
-  I made sure that all tests passed, and the app runs locally (for w/e thats worth).

**Pull request closed without merging**